packages/transformers/babel/src/config.js

```diff
@@ -50,9 +36,7 @@ export async function load(
   );
   let partialConfig = babelCore.loadPartialConfig({
     filename: config.searchPath,
-    cwd: path.dirname(config.searchPath),
-    root: options.projectRoot,
-    babelrcRoots,
```

Comment:
?

Reply:
I believe setting the cwd to the projectRoot makes babel able to find the root config itself rather than needing us to pass them in.